src/utils/enums.py

```diff
@@ -0,0 +1,10 @@
+from foobar.exceptions import InvalidTransition
+
+
+def validate_transition(enum, from_state, to_state):
+    if not hasattr(enum, '_transitions'):
+        raise InvalidTransition
```

Comment:
Yes, will do. Enums contains accessible data, that can be turned into an informal message as well. So it fits good.